.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
```

Comment:
I think we only need the `language` (can remove `dist` and `os`). [example](https://github.com/twilio/twilio-node/blob/main/.travis.yml), [example](https://github.com/yonatankarp/BudgetCalculator-API/blob/master/.travis.yml)